BankSDK/GiniBankSDKPinning/README.md

```diff
@@ -9,33 +9,42 @@
 [![Swift version](https://img.shields.io/badge/swift-5.0-orange.svg)]()
 [![Swift package manager](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)]()
 
-The Gini Bank SDK provides components for capturing, reviewing and analyzing photos of invoices and remittance slips.
+The Gini Bank SDK Pinning provides components for capturing, reviewing and analyzing photos of invoices and remittance slips. The SDK supports certificate pinning.
 
 By integrating this SDK into your application you can allow your users to easily take a picture of a document, review it and get analysis results from the Gini backend.
 
-The Gini Bank SDK can be integrated in two ways, either by using the *Screen API* or the *Component API*. In the Screen API we provide pre-defined screens that can be customized in a limited way. The screen and configuration design is based on our long-lasting experience with integration in customer apps. In the Component API, we provide independent views so you can design your own application as you wish. We strongly recommend keeping in mind our UI/UX guidelines, however.
+The Gini Bank SDK Pinning can be integrated in two ways, either by using the *Screen API* or the *Component API*. In the Screen API we provide pre-defined screens that can be customized in a limited way. The screen and configuration design is based on our long-lasting experience with integration in customer apps. In the Component API, we provide independent views so you can design your own application as you wish. We strongly recommend keeping in mind our UI/UX guidelines, however.
 
-On *iPhone*, the Gini Bank SDK has been designed for portrait orientation. In the Screen API, orientation is automatically forced to portrait when being displayed. In case you use the Component API, you should limit the view controllers orientation hosting the Component API's views to portrait orientation. This is specifically true for the camera view.
+On *iPhone*, the Gini Bank SDK Pinning has been designed for portrait orientation. In the Screen API, orientation is automatically forced to portrait when being displayed. In case you use the Component API, you should limit the view controllers orientation hosting the Component API's views to portrait orientation. This is specifically true for the camera view.
 
 ## Documentation
 
 Further documentation with installation, integration or customization guides can be found in our [website](https://developer.gini.net/gini-mobile-ios/GiniBankSDK/).
 
 ## Example
 
-We are providing the Bank and Health example apps for integrating payment functionality. The Bank app demonstrates how to integrate the Gini Bank SDK. The Health app initiates the payment flow.
+In order to run [the example app](https://github.com/gini/gini-mobile-ios/tree/main/BankSDK/GiniBankSDKExample/GiniBankSDKExample), clone the repo ,open the project file and Resolve Package Versions in Xcode `File->Packages->Resolve Package Versions`.
```

Comment:
I would simply link to the Example section in `BankSDK/GiniBankSDK/README.md`: https://github.com/gini/gini-mobile-ios/tree/main/BankSDK/GiniBankSDK#example

The less duplication the easier it is to maintain the docs.